cactusbot/handlers/events.py

```diff
@@ -102,29 +106,74 @@ def __init__(self, cache_data, api):
     async def on_host(self, packet):
         """Handle host packets."""
 
-        if self.alert_messages["host"]["announce"]:
-            return MessagePacket(
-                self.alert_messages["host"]["message"].replace(
-                    "%USER%", packet.user
-                ))
+        if not self.alert_messages["host"]["announce"]:
+            return
+
+        response = MessagePacket(
+            self.alert_messages["host"]["message"].replace(
+                "%USER%", packet.user
+            ))
+
+        if packet.success:
```

Comment:
Can you put this entire block into a method? Lots of redundant code. :slightly_smiling_face: